app/router/student.py

```diff
@@ -46,10 +47,26 @@ def build_student_and_user_data(student_data):
     return data
 
 
-async def create_school_user_record(data, school_name):
-    school_data = school.get_school(
-        build_request(query_params={"name": str(school_name)})
-    )
+async def create_school_user_record(data, school_name, district, auth_group_name):
+    state = authgroup_state_mapping.get(auth_group_name, "")
+
+    if state:
```

Comment:
even if "state" isn't there.. for whatever reason.. we should still be able to get_school na? with state empty